protocol/0087-VAMM-automated_market_maker.md

```diff
@@ -0,0 +1,288 @@
+  - If other traders trade to move the market mid price to `140` the vAMM has a short position. (<a name="0087-VAMM-006" href="#0087-VAMM-006">0087-VAMM-006</a>)
+  - If other traders trade to move the market mid price to `90` the vAMM has a long position (<a name="0087-VAMM-007" href="#0087-VAMM-007">0087-VAMM-007</a>)
+  - If other traders trade to move the market mid price to `150` the vAMM will post no further sell orders above this price, and the vAMM's position notional value will be equal to `4x` its total account balance. (<a name="0087-VAMM-008" href="#0087-VAMM-008">0087-VAMM-008</a>)
+  - If other traders trade to move the market mid price to `85` the vAMM will post no further buy orders below this price, and the vAMM's position notional value will be equal to `4x` its total account balance.(<a name="0087-VAMM-009" href="#0087-VAMM-009">0087-VAMM-009</a>)
+  - If other traders trade to move the market mid price to `110` and then trade to move the mid price back to `100` the vAMM will have a position of `0`. (<a name="0087-VAMM-010" href="#0087-VAMM-010">0087-VAMM-010</a>)
+  - If other traders trade to move the market mid price to `90` and then trade to move the mid price back to `100` the vAMM will have a position of `0`. (<a name="0087-VAMM-011" href="#0087-VAMM-011">0087-VAMM-011</a>)
+  - If other traders trade to move the market mid price to `90` and then in one trade move the mid price to `110` then trade to move the mid price back to `100` the vAMM will have a position of `0`. (<a name="0087-VAMM-012" href="#0087-VAMM-012">0087-VAMM-012</a>)
+  - If other traders trade to move the market mid price to `90` and then move the mid price back to `100` in several trades of varying size, the vAMM will have a position of `0`. (<a name="0087-VAMM-013" href="#0087-VAMM-013">0087-VAMM-013</a>)
+  - If other traders trade to move the market mid price to `90` and then in one trade move the mid price to `110` then trade to move the mid price to `120` the vAMM will have a larger (more negative) but comparable position to if they had been moved straight from `100` to `120`. (<a name="0087-VAMM-014" href="#0087-VAMM-014">0087-VAMM-014</a>)
+  
+- A vAMM which has been created and is active contributes with it's proposed fee level to the active fee setting mechanism. (<a name="0087-VAMM-015" href="#0087-VAMM-015">0087-VAMM-015</a>)
+- A vAMM's virtual ELS should be equal to the ELS of a regular LP with the same committed volume on the book (i.e. if a vAMM has an average volume on each side of the book across the epoch of 10k USDT, their ELS should be equal to that of a regular LP who has a commitment which requires supplying 10k USDT who joined at the same time as them). (<a name="0087-VAMM-016" href="#0087-VAMM-016">0087-VAMM-016</a>)
+  - A vAMM's virtual ELS should grow at the same rate as a full LP's ELS who joined at the same time. (<a name="0087-VAMM-017" href="#0087-VAMM-017">0087-VAMM-017</a>)
```

Comment:
@TomMcL @wwestgarth after we discussed the scoring and ELS for vAMMs, we agreed that a vAMM would not have an ELS until the next epoch. These two ACs  (16 and 17) either need to be updated, or we need to re-evaluate the current implementation.